app/components/article-featured-video.js

```diff
@@ -32,6 +62,8 @@ export default Component.extend(
 			if (this.player) {
 				this.player.destroy();
 			}
+
+			this.$(window).unbind('scroll');
```

Comment:
This will unbind _all_ on-scroll events. We need to name the function passed to `bind` and use it here as the second param.
Also `bind` and `unbind` are deprecated, see https://api.jquery.com/unbind/